Packs/DeveloperTools/Scripts/Hey/Hey.py

```diff
@@ -0,0 +1,112 @@
+from io import StringIO
+from subprocess import PIPE, Popen
+
+import pandas as pd
+import urllib3
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+# ---------- CLASSES ---------- #
+
+
+class HeyPerformanceResult:
+    def __init__(self,
+                 result: str,
+                 results_map: Optional[str],
+                 t: Optional[str] = None,
+                 c: Optional[str] = None,
+                 n: Optional[str] = None,
+                 **args):
+        self._t = int(t or 20)
+        self._c = int(c or 50)
+        self._n = int(n or 200)
+        c_in_n = self._n / self._c
+        if c_in_n != int(c_in_n):
+            self._n = int(c_in_n) * self._c
+        self._result = result or ''
+        self._ext_results_map = {}
+        if results_map:
+            self._ext_results_map = dict(item.split("=") for item in results_map.split(";"))
+
+    def to_results(self) -> CommandResults:
+        df = pd.read_csv(StringIO(self._result), usecols=['response-time', 'status-code'])
+        requests_num = self._n
+        if len(df) == 0:
+            max_time = 0
+            avg_time = 0
+            min_time = 0
+            total_time = int(self._n / self._c) * self._t
+            successful_responses = 0
+        else:
+            response_times = df['response-time']
+            max_time = max(response_times)
+            min_time = min(response_times)
+            avg_time = response_times.mean()
+            total_time = response_times.sum() / int(self._c)  # not 100% accurate
+            status_codes = df.get('status-code')
+            successful_responses = len(tuple(code == 200 for code in status_codes))
+        outputs = {
+            "TimeoutPerRequest": self._t,
+            "Concurrency": self._c,
+            "Requests": requests_num,
+            "SlowestTime": max_time,
+            "FastestTime": min_time,
+            "AverageTime": avg_time,
+            "TotalTime": total_time,
+            "SuccessfulResponses": successful_responses
+        }
+        outputs.update(self._ext_results_map)
+        return CommandResults(outputs=outputs, outputs_prefix="Hey")
+
+
+# ---------- HELPER FUNCTIONS ---------- #
+
+def run_command(command: str) -> str:
+    """Run a bash command in the shell.
+
+    Args:
+        command (string): The string of the command you want to execute.
+
+    Returns:
+        string. The output of the command you are trying to execute.
+    """
+    p = Popen(command.split(), stdout=PIPE, stderr=PIPE, universal_newlines=True)
+    output, err = p.communicate()
```

Comment:
don't think this should fail if err has output. it should fail based upon the return code. Anyway, don't use Popen. You should use subrocess.check_output which is a higher level api. See: https://docs.python.org/3/library/subprocess.html#subprocess.check_output